backend/entityservice/api_def/openapi.yaml

```diff
@@ -0,0 +1,1078 @@
+      discriminator:
+        propertyName: state
+      properties:
+        time_added:
+          type: string
+          format: 'date-time'
+        state:
+          type: string
+          description: What state is the run in
+          enum:
+            - created
+            - queued
+            - running
+            - completed
+            - error
+        stages:
+          type: number
+          format: int
+          minimum: 1
+          description: total number of stages for this run
+        current_stage:
+          $ref: '#/components/schemas/RunStage'
+      required:
+        - time_added
+        - state
+        - stages
+        - current_stage
+
+    completed:
+      allOf:
+        - $ref: '#/components/schemas/RunStatus'
+        - type: object
+          properties:
+            time_started:
+              type: string
+              format: 'date-time'
+            time_completed:
+              type: string
+              format: 'date-time'
+          required:
+            - time_started
+            - time_completed
+
+    error:
+      description: Describes the errored Run. Must contain a 'message', can provide more detail in 'detail'.
+      allOf:
+        - $ref: '#/components/schemas/RunStatus'
+        - type: object
+          properties:
+            message:
+              type: string
+              description: short error message
+            detail:
+              type: string
+              description: detailed error message
+          required:
+            - message
+
+    queued:
+      allOf:
+        - $ref: '#/components/schemas/RunStatus'
+        - type: object
+          properties:
+            time_started:
+              type: string
+              format: 'date-time'
+
+    started:
+      allOf:
+        - $ref: '#/components/schemas/queued'
+
+    running:
+      allOf:
+        - $ref: '#/components/schemas/RunStatus'
+        - type: object
+          properties:
+            time_started:
+              type: string
+              format: 'date-time'
+          required:
+            - time_started
+
+    RunStage:
+      description: describes the current stage of a run
+      type: object
+      properties:
+        number:
+          type: number
+          minimum: 1
+          description: the number of the current stage, first stage has number 1
+        description:
+          type: string
+          description: describes the current stage in human readable form
+        progress:
+          $ref: '#/components/schemas/RunProgress'
+      required:
+        - number
+
+    CLKUpload:
+      description: Array of this party's Bloom Filters
+      type: object
+      required: [clks]
+      properties:
+        clks:
+          type: array
+          items:
+            type: string
+            format: byte
+            description: Base64 encoded CLK data
+
+    CLKnBlockUpload:
+      description: Array of this party's Bloom Filters including blocking information
```

Comment:
This is an object containing a single element which is an array. Do we need/want the extra layer just so you can say `clknblocks`?

Reply:
With the clks only, we upload a json of the form {"clks": [...]}.
Therefore I thought that for better separation, we will give the new format a new key.
That simplifies figuring out the format on the receiving end.
I also like to have the clks under a key and not top level, such that we can easily add meta info without breaking anything.

Reply:
It would still be nice for the description to be accurate - i.e. `object` rather than `Array`